app/ui.js

```diff
@@ -180,7 +180,7 @@ const UI = {
         UI.initSetting('shared', true);
         UI.initSetting('view_only', false);
         UI.initSetting('show_dot', false);
-        UI.initSetting('path', 'websockify');
+        UI.initSetting('path', window.location.pathname.substr(1) + '/../websockify');
```

Comment:
This is a bit roundabout way of getting to where we want¹. I'd like to see something where we construct the sane, final path and hand off here.

¹ And `substr()` is also deprecated. :)

Reply:
I've updated the usage of the deprecated `substr()` method.
And I agree, not sure if this is the best method, but the main idea is to get the ball rolling because this issue seems to keep being brought up.

Reply:
Completely agree with @vsilvar 
We are facing this issue since years ago